lib/hal/video.c

```diff
@@ -31,7 +31,7 @@
 
 
 unsigned char*	_fb;
-DWORD			dwEncoderSettings 	= 0;
+DWORD			dwEncoderSettings	= 0;
```

Comment:
Accidental change? This file uses bad tabs anyway, so it's doomed to fail, if this is for alignment.

Reply:
Fixed. :+1: